packages/web/src/client/components/Video/VideoPlayer.tsx

```diff
@@ -134,3 +144,31 @@ const Video = styled('video', {
   width: '100%',
   height: '100%',
 })
+
+const Control = styled('button', {
+  bottom: '16px',
+  right: '16px',
+  background: 'white',
+  width: '40px',
+  height: '40px',
+  position: 'absolute',
+  display: 'flex',
```

Comment:
Does this need to be flex?

Reply:
@joshuaellis I want to center the image `span` inside. So, I was planning this to be `flex`, so I can apply `justifyContent: 'center'` in the children.

Reply:
Justify content is for the children I believe not the children.